corehq/apps/geospatial/templates/gps_capture.html

```diff
@@ -26,14 +26,16 @@ <h3 class="panel-title">
             </h3>
         </div>
         <div class="panel-body">
-            <div class="row">
-                <div class="col-sm-6">
-                    <search-box data-apply-bindings="false"
-                                params="value: query,
-                                        action: function() { goToPage(1); },
-                                        placeholder: '{% trans_html_attr "Search..." %}'"></search-box>
+            {% if data_type == 'user' %}
```

Comment:
Since we have a query filter within the filters, this is not needed for cases anymore